### PR TITLE
Added requirements, readme

### PR DESCRIPTION
**Pull request by sananand007**
Added README and requirements

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+certifi==2019.6.16
```
Comment (weihesdlegend) [inline requirements.txt]: Do we need all these or that guy’s requirement.txt is enough?

Comment (sananand007) [reply requirements.txt]: These are all pip freeze , they are auto generated.its same as the other guys .

**Comment by sananand007** [comment]
@weihesdlegend Changes are made for merge

**Comment by sananand007** [comment]
yes we would need all the extra packages for the branches and then merge into release

**Pull request merged**